### PR TITLE
La 384 cleanups

### PR DESCRIPTION
**Pull request by evrardjp**
Issue: [LA-384](https://rpc-openstack.atlassian.net/browse/LA-384)

### CODE REVIEW

File: scripts/leapfrog/README.md [hunk 1]
```diff
@@ -98,6 +98,8 @@ If a leapfrog upgrade is interrupted, it can be resumed. Each major step of the
 leapfrog upgrade process creates a marker file, which will be used to skip
 completed tasks on subsequent runs.
 
+To resume, the deployer can just run scripts/leapfrog/ubuntu14-leapfrog.sh.
```
Comment (hughsaunders) [inline scripts/leapfrog/README.md]: s/, the deployer can just//

**Comment by evrardjp** [comment]
@hughsaunders Adapted, could you have a look at the english again?

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift -- tempest issue server got down.

File: scripts/leapfrog/README.md [hunk 2]
```diff
@@ -98,6 +98,8 @@ If a leapfrog upgrade is interrupted, it can be resumed. Each major step of the
 leapfrog upgrade process creates a marker file, which will be used to skip
 completed tasks on subsequent runs.
 
+To resume, run scripts/leapfrog/ubuntu14-leapfrog.sh.
+
```
Comment (mancdaz) [inline scripts/leapfrog/README.md]: To resume, simply run ```scripts/leapfrog/ubuntu14-leapfrog.sh``` again

**Comment by evrardjp** [comment]
See https://github.com/rcbops/rpc-openstack/pull/2444

**Pull request closed without merging**